src/test/resources/conf/version.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version":"1.5.0",
+  "url":"http://devonfw.github.io/download/devcon/devcon.jar",
+  "devondist_file_id": "frs65334",
+  "devondist_file_linux_id": "frs65335",
+  "oaspide_file_id": "frs54315",
+  "oasp4js_ang1_id": "frs51721",
+  "oasp4js_ang2_id": "frs51961",
+  "oasp_template_version": "2.6.0"
```

Comment:
maybe better to rename it already or provide two versions here.
oasp4j_template_version / devon4j_template_version. So what is the way to go here is not interesting for me but it should be future proof as of the renaming.

Reply:
IMHO, as devcon is a devonfw tool we should not mix versions. If devon4j is going to be the version that is going to come with the new distribution we will need again to refactor devcon and only refer to the new `devon4j_template_version`.